types/jotform-css.js/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
This file should only `"extends": "dtslint/dt.json"` for new packages, you can disable rules in source code

Reply:
I currently get this error which makes no sense to me `In /Users/spurrymoses/code/DefinitelyTyped/types/jotform-css.js: Expected file 'jotform-css.js-global.test.ts' to be named jotform-css.js-tests.ts`

Given that I'm following instructions from further down the page, 2 have 2 different test files.
 `Name one YourLibraryName-global.test.ts and the other YourLibraryName-module.test.ts`

That instruction is at least wrong on the account that the file must actually be named `...tests` ie "with an added s" or another error will be raised.

Reply:
Even if I add the file `jotform-css.js-tests.ts` - the error persists.

Reply:
@andy-ms

Reply:
Sorry, I worked out I needed to add the folder "tests", I've deleted my comments to avoid clutter